src/h3lib/lib/coordijk.c

```diff
@@ -346,8 +346,8 @@ H3Error _upAp7Checked(CoordIJK *ijk) {
     }
 
     // TODO: Do the int math parts here in long double?
```

Comment:
I'm a little late here, but it looks like there are some TODO comments that are now out of date